src/coreclr/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -6633,11 +6639,12 @@ const WCHAR* MethodContext::repGetStringConfigValue(const WCHAR* name)
     if (nameIndex == -1) // config name not in map
         return nullptr;
 
-    int            resultIndex = GetStringConfigValue->Get(nameIndex);
-    const WCHAR* value       = (const WCHAR*)GetStringConfigValue->GetBuffer(resultIndex);
+    AssertKeyExists(GetStringConfigValue, nameIndex, "");
```

Comment:
Nit: took me a few seconds to understand that the index from `Contains` should be used in`_Item Get(_Key key) const` and not in `_Key GetKey(int index) const`, would it be better to rename `int nameIndex` to `DWORD key`?

Reply:
This is a case where the names in "rep" match the names in "rec", which actually make sense. So I think I'll leave them.